libs/coin-bitcoin/src/wallet-btc/crypto/bitcoincash.ts

```diff
@@ -1,5 +1,5 @@
 import bchaddr from "bchaddrjs";
-import { toOutputScript } from "bitcoinjs-lib/src/address";
+import { address as btcAddress } from "bitcoinjs-lib";
```

Comment:
we can see [here](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/src/index.js) that index acts as a [barrel](https://flaming.codes/en/posts/barrel-files-in-javascript/) so it should be fine editing the import. 

```
const address = require('./address');
exports.address = address;
```

Reply:
I don't get it. Do you want to use `require` instead? `import "bicoinjs-lib/address"` seems not to work (I think the lib miss some declaration in it package.json to work properly).

Reply:
No :smile: the snippet is from bitcoinjs-lib, `index` exports `src/address`